src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// categories.js
+
+// Actions
+const CHECK_CATEGORIES = 'CHECK_CATEGORIES';
+
```

Comment:
The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so Kindly add them following the ducks pattern. :+1: it will be something like: :arrow_down:

```
const CHECK_CATEGORIES = bookstore/categories/CHECK_CATEGORIES
```